.gitignore

```diff
@@ -47,6 +47,7 @@ npm-debug.log
 /tsd_typings/
 
 # Dist #
+/lint
```

Comment:
I've noticed the "lint" directory appearing and being untracked before, so this seems like a good idea, but maybe it should be pulled out of this PR?

Reply:
I agree. This should be separate.